.github/workflows/integration_tests_wsl.yml

```diff
@@ -0,0 +1,78 @@
+name: integration-tests
+
+on:
+  push:
+    branches: [ "main" ]
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - '**/*.md'
+
+permissions:
+  contents: read
+
+jobs:
+  example-counter-wsl:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Clone Reflex Examples Repo
+        uses: actions/checkout@v3
+        with:
+          repository: reflex-dev/reflex-examples
+          path: reflex-examples
+
+      - uses: Vampire/setup-wsl@v2
+
+      - name: Install Python
+        shell: wsl-bash {0}
+        run: |
+          apt update
+          apt install -y python3 python3-pip curl dos2unix zip unzip
+
+      - name: Install Poetry
+        shell: wsl-bash {0}
+        run: |
+          curl -sSL https://install.python-poetry.org | python3 -
+          # Symlink seems easier to make work than persisting PATH changes, in WSL
+          ln -s /root/.local/bin/poetry /usr/local/bin/poetry
+
+      - name: Install reflex deps
+        shell: wsl-bash {0}
+        run: |
+          poetry install
+
+      - name: Install requirements for counter example
+        working-directory: ./reflex-examples/counter
+        shell: wsl-bash {0}
+        run: |
+          poetry run pip install -r requirements.txt
+      - name: Check export --backend-only before init for counter example
+        working-directory: ./reflex-examples/counter
+        shell: wsl-bash {0}
+        run: |
+          poetry run reflex export --backend-only
+      - name: Check run --backend-only before init for counter example
+        shell: wsl-bash {0}
+        run: |
+          dos2unix scripts/integration.sh
+          poetry run bash scripts/integration.sh ./reflex-examples/counter dev 8001 --backend-only --backend-port 8001
+      - name: Init Website for counter example
+        working-directory: ./reflex-examples/counter
+        shell: wsl-bash {0}
+        run: |
+          poetry run reflex init --loglevel debug
+          # Symlink seems easier to make work than persisting PATH changes, in WSL
+          ln -s ~/.reflex/.bun/bin/bun /usr/local/bin/bun
+     # TODO not working - to be figured out.  Hangs just after "automatically rendered as static HTML"
```

Comment:
Hangs in the middle of `npm export`... could not make quick progress. Parking this for now, in interest of getting SOME coverage out there for WSL.

Could not repro in my Azure box.